docs/podman-create.1.md

```diff
@@ -654,6 +654,40 @@ change propagation properties of source mount. Say `/` is source mount for
 To disable automatic copying of data from the container path to the volume, use
 the `nocopy` flag. The `nocopy` flag can be set on bind mounts and named volumes.
 
+**--volumes-from**[=*[CONTAINER[:OPTIONS]]*]
+
+Mount volumes from the specified container(s).
+The `OPTIONS` are a comma delimited list and can be:
+
+* [rw|ro]
+* z
+
+Mounts already mounted volumes from a source container onto another
+container. You must supply the source's container-id or container-name.
+To share a volume, use the --volumes-from option when running
+the target container. You can share volumes even if the source container
+is not running.
+
+By default, podman mounts the volumes in the same mode (read-write or
+read-only) as it is mounted in the source container. Optionally, you
+can change this by suffixing the container-id with either the :ro or
+:rw keyword.
```

Comment:
I'd rather not include the colons here; that's part of a higher level than the `ro` and `rw` options themselves.  And I think this paragraph could be inlined in the *OPTIONS* list above:

```markdown
*OPTIONS* is a comma-delimited list with the following available elements:

* `rw` or `ro`.  By default, ...
* `z`.  Labeling systems like SELinux...
```

Reply:
Fixed. I left the paragraphs the way they were as it was looking weird to me when adding it inline to the options.